frontend/src/stores/roms.ts

```diff
@@ -15,48 +16,44 @@ export type DetailedRom = DetailedRomSchema;
 
 export default defineStore("roms", {
   state: () => ({
-    _platformID: 0,
-    _all: [] as SimpleRom[],
+    currentPlatform: null as Platform | null,
```

Comment:
there's a reason i used platformID internally, instead of the platform... if i remember ill update this comment

Reply:
i tried every possible way to navigate through romm and coudn't find any issue using the platform

Reply:
And of course checked every place where we were using _platformID and no issues at all